packages/openmrs/src/Utils.js

```
@@ -15,6 +15,16 @@ export const prepareNextState = (state, response, callback = s => s) => {
   return callback(nextState);
 };
 
+export async function fetchAndLog(method, url, opts) {
+  try {
+    const response = await commonRequest(method, url, opts);
+    logResponse(response);
+    return response;
+  } catch (err) {
+    return err;
```

I think we should let this throw?